.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+on:
```

Comment:
Maybe this file could be renamed to `test.yml` as we are going to have multiple workflows in this folder.

Reply:
In our team we have the whole lint + test + build set in one file, but we can also separate those? I personally maybe feel that there is "one file for one thing" (like this would be the when-pr-is-made-file), but that is 100% a matter of opinion.

Reply:
Sure we should have those in the same file, but since we have also other workflows (with different triggers) that can be conceptually part of the "CI" as well, I would choose a different name than CI 🤔  However, if you feel that this is the best choice, you can leave it as is.